### PR TITLE
Prioritize DjangoModelPermissionsOrAnonReadOnly when checking V3APIPermission 

### PR DESCRIPTION
**Pull request by albertisfu**
Fixes a vulnerability in API endpoints that previously relied solely on the default `DjangoModelPermissionsOrAnonReadOnly` permission class. 

This class is now included within `V3APIPermission`, ensuring its check is prioritized before other permission checks.

### CODE REVIEW

File: cl/disclosures/api_views.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from rest_framework import viewsets
+from rest_framework.permissions import DjangoModelPermissionsOrAnonReadOnly
```
Comment (mlissner) [inline cl/disclosures/api_views.py:2]: This needed here?

**Review by mlissner — APPROVED** [review]

**Comment by mlissner** [comment]
The approach looks good to me. If tests pass, and Eduardo agrees, we're off to the races.

**Review by ERosendo — APPROVED** [review]
@albertisfu Your code looks good. I see you've updated all the classes that were previously using the default permissions ( I compared this PR to https://github.com/freelawproject/courtlistener/pull/4579/) :shipit:

**Comment by mlissner** [comment]
Thank you both!

**Pull request merged**